src/pyodide/make_snapshots.py

```diff
@@ -23,7 +23,7 @@
     {requirements}
   ],
   compatibilityDate = "2023-12-18",
-  compatibilityFlags = ["python_workers", {compat_flags}],
+  compatibilityFlags = ["python_workers", "python_no_global_handlers", {compat_flags}],
```

Comment:
I wonder if we should also increment the compat date on these while we're at it

Reply:
I think that will work. I've seen workerd resetting a compat date that's in the future to its own release date with only warning.

Reply:
It does not work with future dates.